src/client/src/App/Trades/TradesGrid/TradesGrid.tsx

```diff
@@ -144,26 +144,13 @@ export const TradesGridInner: React.FC<{
 export const TradesGrid: React.FC = () => {
   const trades = useTableTrades()
   const highlightedRow = useTradeRowHighlight()
-
-  const tryBroadcastContext = (symbol: string) => {
-    const context = {
-      type: "fdc3.instrument",
-      id: { ticker: symbol },
-    }
-
-    if (window.fdc3) {
```

Comment:
@taylor-adaptive can you please review and ensure this doesn't break anything in workspace? thanks!